inference-engine/tests/functional/inference_engine/transformations/fq_mul_fusion_test.cpp

```diff
@@ -0,0 +1,126 @@
+        std::tie(data_shape, in_shape, out_shape, mul_const_shape) =
+            this->GetParam();
+
+        const auto data = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, data_shape, {0.0f});
+        const auto in_low = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, in_shape, {-0.5f});
+        const auto in_high = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, in_shape, {0.5f});
+        const auto out_low = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, out_shape, {0.0f});
+        const auto out_high = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, out_shape, {100.0f});
+        const auto fq = std::make_shared<ngraph::opset4::FakeQuantize>(
+            data, in_low, in_high, out_low, out_high, 255);
+
+        const auto mul_value = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, mul_const_shape, {3.14f});
+        const auto mul = std::make_shared<ngraph::opset4::Multiply>(fq, mul_value);
+
+        m_function = std::make_shared<ngraph::Function>(
+            ngraph::OutputVector{mul}, ngraph::ParameterVector{}, "FQMulFusion");
+
+        const auto expected_data = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, data_shape, {0.0f});
+        const auto expected_in_low = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, in_shape, {-0.5f});
+        const auto expected_in_high = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, in_shape, {0.5f});
+        const auto expected_out_low = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, out_shape, {0.0f});
+        const auto expected_out_high = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, out_shape, {100.0f});
+
+        const auto expected_mul_value = ngraph::opset4::Constant::create(
+            ngraph::element::Type_t::f32, mul_const_shape, {3.14f});
+        const auto expected_mul_out_low =
+            std::make_shared<ngraph::opset4::Multiply>(expected_out_low, expected_mul_value);
+        const auto expected_mul_out_high =
+            std::make_shared<ngraph::opset4::Multiply>(expected_out_high, expected_mul_value);
+        const auto expected_fq = std::make_shared<ngraph::opset4::FakeQuantize>(
+            expected_data, expected_in_low, expected_in_high, expected_mul_out_low,
+            expected_mul_out_high, 255);
+
+        m_expected_function = std::make_shared<ngraph::Function>(
+            ngraph::OutputVector{expected_fq}, ngraph::ParameterVector{}, "FQMulFusion_expected");
+    }
+
+  std::shared_ptr<ngraph::Function> m_function;
+  std::shared_ptr<ngraph::Function> m_expected_function;
+};
+
+TEST_P(FQMulFusion, ExpectFusion) {
+  ngraph::pass::Manager manager;
+  manager.register_pass<ngraph::pass::FakeQuantizeMulFusion>();
```

Comment:
Please also register https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/inference_engine/transformations/convert_pad_to_group_conv.cpp#L39 in the beginning and then after `run_passes` call https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/inference_engine/transformations/convert_pad_to_group_conv.cpp#L44